crates/fluvio-stream-dispatcher/src/dispatcher/k8_dispatcher.rs

```diff
@@ -187,14 +187,15 @@ where
     ///
     #[instrument(skip(self))]
```

Comment:
bump up crate version

Reply:
I think 0.6.5 [hasn't been published to crates.io yet](https://crates.io/crates/fluvio-stream-dispatcher/versions) so I don't think it's needed?

Reply:
yup.